rollup.config.js

```diff
@@ -1,20 +1,39 @@
+import babel from '@rollup/plugin-babel';
 import replace from '@rollup/plugin-replace';
-import commonjs from '@rollup/plugin-commonjs';
 
-export default {
+const commonConfig = {
 	input: 'index.js',
-	output: {
-		'file': __dirname + '/dist/memize.js',
-		format: 'iife',
-		name: 'memize',
-	},
```

Comment:
I removed this format, though if you want to keep supporting it I can add it back in, it really doesn't take much to build all the different formats.

Reply:
Nah, I think it's fine to remove this output format. I'd probably consider it a breaking change though.